scripts/proteinInteractionEBI/README.md

```diff
@@ -0,0 +1,62 @@
+# Scripts for importing ontology dataset from the European Bioinformatics Institute (EMBL-EBI)
+
+This directory stores all scripts used to import datasets from the European Bioinformatics Institute (EMBL-EBI). 
+Here we only import the three subsets of the ontologies: "interaction detection method", "interaction type" and "database citation", which are commonly used in protein-protein interactions. 
+
+## Usage
+
+To generate 'BioOntologySchemaEnum.mcf' which contains all the schemas and "psimi2dcid.txt" which contains paired PSI-MI identifiers and DCID (../proteinInteractionMINT/parseMINT.py needs to use this file to refer to the corresponding Enum instance by DCID), run:
+
+```bash
+python3 parseEBI.py mi.owl 
+```
+
+If new reference sources which are not properties in dcs occur, 'BioEBINewSource.txt' containing such information will be generated as well. If this is the case, new properties should be created for the new source (similar to property "dcs:pubMedID"), and the script should handle the new source corespondingly.
+
+## Database format
+
+Data is available for download from
+http://psidev.info/groups/controlled-vocabularies. It can also be browsed at https://www.ebi.ac.uk/ols/ontologies/mi. 
+The ontologies dictionary has a tree structure. Note here that one parent node can have multiple child nodes and one child node can have multiple parent nodes as well.
+
+![Tree Structure](./graph/ontologyTree.png)
+
+![Multiple Parent Node](./graph/multipleParent.png)
```

Comment:
Is this something you created or downloaded from ebi? Does it work to link to the source instead of checking in?

Reply:
These are two screenshots showing the data format. I can put links here instead.